code/modules/mob/living/carbon/xenomorph/castes/boiler/abilities_boiler.dm

```diff
@@ -181,7 +185,7 @@ GLOBAL_LIST_INIT(boiler_glob_image_list, list(
 
 /datum/action/xeno_action/activable/bombard/get_cooldown()
 	var/mob/living/carbon/xenomorph/boiler/X = owner
-	return X.xeno_caste.bomb_delay - ((X.neuro_ammo + X.corrosive_ammo) * X.xeno_caste.ammo_multiplier)
+	return X.xeno_caste.bomb_delay - ((X.neuro_ammo + X.corrosive_ammo) * (BOILER_BOMBARD_COOLDOWN_REDUCTION SECONDS))
```

Comment:
X.xeno_caste.ammo_multiplier ??

Reply:
Thats what it used to be. It's being changed to be a macro